Library/Homebrew/cmd/update-bash.sh

```diff
@@ -271,7 +287,6 @@ EOS
       odie "Git must be installed and in your PATH!"
     fi
   fi
-  hash -p "$(cd "$(dirname "$(which_git)")" && pwd -P)/git" git
```

Comment:
You're removing this but you're not actually resolving the full path at any point so it'll break when you change directory.